amaznot/app/Http/Controllers/SearchController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Search;
+use Illuminate\Database\QueryException;
+
+class SearchController extends Controller
+{
+    public function search(Request $request, $search = "")
```

Comment:
If you're if condition returns false and there is no exception thrown, this controller method returns nothing at all. In fact, this seems to be the default behaviour, as going to /search/anything simply returns a blank page.

Reply:
Oh damn, I'll fix it as soon as I can and send a new one